### ISSUE TITLE
Change Account.Msg.TransferSucceed to append transfer

### ISSUE THREAD

**Issue by lucca65** [BeSpiral/app#2]
After #1, we need to make the frontend app receive those transactions and append them to Balance and Communities

Related to: #1, [platform#2](https://github.com/BeSpiral/bespiral-platform/issues/2)

**Comment by lucca65** [comment]
No longer relevant
